src/models/googleUser.ts

```diff
@@ -0,0 +1,15 @@
+interface googleUser {
+_id: string;
+iss: string,
+nbf: number;
+aud: string;
+sub: string;
+hd: string;
+email: string;
+email_verified: boolean;
+azp: string;
+name: string;
+picture: string;
+}
+
```

Comment:
i like the interfaces -- however, i think the linter is broken -- these keys shouldnt all be without an indent -- try to fix the linter on the next PR